### PR TITLE
Visual update to Cedar documentation

### PR DESCRIPTION
**Pull request by AMZ-brandon**
This PR updates the Cedar documentation to dark mode with Amazon fonts.

Preview at https://amz-brandon.github.io/

### CODE REVIEW

File: docs/_sass/color_schemes/awsstyle.scss [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+$content-width: 30rem !default;
+$header-height: 5rem !default;
+
+
+
+//############ Classes ############
+
+ body {
+	//overflow-y:none !important;
+	 position:fixed;
```
Comment (Swolebrain) [inline docs/_sass/color_schemes/awsstyle.scss:41]: why position fixed?

File: docs/_sass/color_schemes/awsstyle.scss [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	 font-family: "Amazon Ember";
```
Comment (Swolebrain) [inline docs/_sass/color_schemes/awsstyle.scss:42]: we need another !important hack here so the body is actually in ember, or we can move this to a separate selector with higher specificity like `body *`

Comment (AMZ-brandon) [reply docs/_sass/color_schemes/awsstyle.scss]: Done, thanks!

**Review by mwhicks1 — APPROVED** [review]
Not a frontend expert, so my review is on the result -- looks great!

**Pull request merged**